.github/workflows/CI.yml

```diff
@@ -33,37 +33,37 @@ jobs:
         with:
           repository: 'opensearch-project/OpenSearch'
           path: OpenSearch
-          ref: 'main'
+          ref: '1.x'
       - name: Build OpenSearch
         working-directory: ./OpenSearch
-        run: ./gradlew publishToMavenLocal -Dbuild.version_qualifier=beta1 -Dbuild.snapshot=false
+        run: ./gradlew publishToMavenLocal -Dbuild.version_qualifier=rc1 -Dbuild.snapshot=false
 
       # dependencies: common-utils
       - name: Checkout common-utils
         uses: actions/checkout@v2
         with:
-          ref: '1.0.0-beta1'
+          ref: 'main' # TODO: update to the right branch name once it's ready. e.g. 1.x
```

Comment:
We should add instructions for our branching strategy for plugins in the plugins repo. @dblock 
I can take care of it.

Reply:
Please do.

Reply:
Issue: https://github.com/opensearch-project/opensearch-plugins/issues/35